packages/jmix-addon-charts/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import { BarChart } from "./component/BarChart";
+import { PieChart } from "./component/PieChart";
+import { LineChart } from "./component/LineChart";
+
+export const chartAddon = {BarChart, LineChart, PieChart};
```

Comment:
Why do we export them like this (putting everything inside `chartAddon`)? It is not convenient to use:

```
import {chartAddon} from "@haulmont/jmix-addon-charts";

<chartAddon.BarChart data={data} height={500} width={500} />
```